composer.json

```diff
@@ -75,7 +75,7 @@
         "symfony/css-selector": "^4.4",
         "symfony/filesystem": "^4.4",
         "symfony/maker-bundle": "^1.17",
-        "symfony/phpunit-bridge": "^5.1",
+        "symfony/phpunit-bridge": "^5.1.1",
```

Comment:
Is this the correct way to require minimal 5.1.1? We need this because:

[PhpUnitBridge] Fix ExpectDeprecationTrait::expectDeprecation() conflict.
https://github.com/symfony/phpunit-bridge/compare/v5.1.0...v5.1.1

Reply:
Yes it is